cmd/zetatool/config/config.go

```diff
@@ -14,18 +14,19 @@ var AppFs = afero.NewOsFs()
 const (
 	FlagConfig         = "config"
 	defaultCfgFileName = "zetatool_config.json"
+	FlagDebug          = "debug"
 )
 
 func TestnetConfig() *Config {
```

Comment:
This file should be updated with working urls , for a seamless operation. 
I have added mostly reliable public apis. 
Rate limiting should not be, mostly an internal tool . If we plan to use this as an API , a config should be provided